client/dashboard/sites/backups/test/index.test.tsx

```diff
@@ -19,6 +19,10 @@ jest.mock( '../../../app/auth', () => ( {
 	} ),
 } ) );
 
+jest.mock( '@wordpress/react-i18n', () => ( {
```

Comment:
This is due to Breadcrumbs dependency of `@wordpress/react-i18n`. Alternatively, we can mock Breadcrumbs to return null.